lib/release_notes_bot_web/controllers/captains_controller.ex

```diff
@@ -124,5 +67,7 @@ defmodule ReleaseNotesBotWeb.CaptainsController do
       _ ->
         nil
     end
+
+    conn |> Plug.Conn.send_resp(200, [])
```

Comment:
Dont use a pipeline for only a single function call, just use the argument directly.

    Plug.Conn.send_resp(conn, 200, [])

and, in this case, if we are sending no content should we be sending a `204 No Content` response code instead of `200 Success`?